infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SenderBufferManager.java

```diff
@@ -260,6 +294,17 @@ public LogReplicationEntryMsg resend() {
         return ack;
     }
 
+    private void activateBackpressure() {
+        log.info("Log entry ACKs timing out on sink, enabling back pressure for retries.");
+        isBackpressureActive.set(true);
+    }
+
+    private void deactivateBackpressure() {
```

Comment:
We should also deactivate the back pressure when fsm transisitons out of log-entry sync.